testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/CliCompletionTestCase.java

```diff
@@ -1158,10 +1158,12 @@ public void testCommandsCompletion2() throws Exception {
                 List<String> candidates = new ArrayList<>();
                 ctx.getDefaultCommandCompleter().complete(ctx, cmd,
                         cmd.length(), candidates);
-                assertTrue(candidates.toString(), candidates.size() == 1);
+                assertTrue(candidates.toString(), candidates.size() == 2);
```

Comment:
@ChristinaDsl , why do we have 2 elements there? We should have namespaces only.
For example if you type: read-attribute name<TAB>, you should get read-attribute namespaces, I can see an added "" character that breaks this completion.

Reply:
If you type read-attribute name, I thought that it would be right to have two choices:
Either you are allowed to leave it as it is "read-attribute name" or
End up in "read-attribute namespaces"
That is why I added empty space as another possible completion.
If there is no such a case I will erase it.

Reply:
@ChristinaDsl , Thank-you, I see, I feel that we should keep the original behavior, this change would be outside the scope of the fix.

Reply:
Ok, I will fix it then, thank you too.

Reply:
@jfdenise 
With my existed modifications, after chunks which has Model Type:
list
string or 
object 
it is possible to add a property separator (if completed chunk concerns operation) or "" (if it is a command). Thus, because "name" lies to the case of a completed string and command, in the same time, an empty space is proposed. This new addition concerns the general case of proposing a comma or empty space after a completed string (which is right behavior) and if it's changed then the issue should be resolved in a completely other way.

Reply:
@ChristinaDsl , I am not really following your comment there, could you give examples of what the problem is ? Thank-you.

Reply:
The part of existing code (not mine), which says that after a completed chunk, in possible completions should exist "," or "",  whether chunk is an operation or command accordingly, covers this case also( with "name" chunk). So, when "name" occurs, which is a command (and string in the same time), it is added in candidates the "".
Another example is typing "read-attribute organization" after hitting tab it proposes the empty space "", which, before my modifications, user had to type it by his own to continue.
Also after "read-attribute release-version" or "read-attribute server-state", "read-attribute uuid" and many others, happens the same thing.
Because "name" is a complete chunk by its own and  belongs to the same category as all of above which follows the "read-attribute" it has the same behaviour. Same happens with chunks following "read-operation".

Reply:
@jfdenise Does this makes sense? Would you like anything more?

Reply:
@ChristinaDsl , I did a test, although read-attribute organization works as you describe, read-attribute management-minor-version doesn't automatically inline the " ".

Reply:
@jfdenise You are right. management-minor-version such as management-micro-version and management-major-version in chunkDescr has ModelType "INT". My modifications apply to three types: List, Object and String(in this last type is "name" included). Would you like me to add INT too?

Reply:
@jfdenise I didn't limit it on purpose. Very few cases are on the INT type,  while most of them are Strings. I was mainly testing it in Strings and that is why I did not take into consideration INT. I need to search it further in order to see if would be unwanted side effects in applying it to INT.

Reply:
@jfdenise What do you believe I need to do next?

Reply:
Do you plan to apply to INT?